File: typescript/src/PlanetTS1/Model/Enemy/Units/EBuilder.ts
import { ALocation as ALocation } from "../../Utilities"

export class EBuilder extends Builder {
    loc : ALocation
    use : boolean
    drive : boolean
    owner : number
    icon : string
    constructor(location : ALocation) {
        super()
        this.loc = location
        this.drive = true
        this.owner = 2
        this.icon = "eB"
    }
}

File: dotnet/src/Model/Enemy/Units/EBuilder.cs
using System;

namespace PlanetCS1 {
    class EBuilder : Builder {
        public EBuilder(Location location) {
            this.loc = location;
            this.use = true;
            this.drive = true;
            this.owner = 2;
            this.icon = "eB";
        }
    }
}